src/app/asign-family/asign-family.component.scss

```diff
@@ -103,4 +103,21 @@ h5 {
 
 ::ng-deep .mat-form-field-infix {
   width: auto !important;
-}
\ No newline at end of file
+}
+
+.search-autocomplete-option {
+  position: absolute;
```

Comment:
I couldn't find a way of extending the span with classname of "mdc-list-item__primary-text" so I had to do this ugly absolute thing :( 
Lmk if you have any other way